### PR TITLE
Add corner case when we get avi controller version

### PR DESCRIPTION
**Pull request by chenlin07**
- Add corner case when we get avi controller version
- Add unit test for config_envvar.go file

Signed-off-by: Chen Lin <linch@vmware.com>

**What this PR does / why we need it**:
- We need to consider the corner case that we set avi controller version parameter but the value is empty. In this case, we want to return the default avi controller version.
- We need unit test for config_envvar.go file
**Which issue(s) this PR fixes**:

Fixes #

**Describe testing done for PR**:

**Special notes for your reviewer**:

**Release note**:

```release-note

```
**New PR Checklist**

- [ ] Ensure PR contains only public links or terms
- [ ] Use good commit [messages](https://github.com/vmware-tanzu/tanzu-framework/blob/main/CONTRIBUTING.md)
- [ ] Squash the commits in this branch before merge to preserve our git history
- [ ] If this PR is just an idea or POC, use a [Draft PR](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests) instead of a full PR
- [ ] Add appropriate [labels](https://github.com/vmware-samples/load-balancer-operator-for-kubernetes/labels) according to what type of issue is being addressed.

### CODE REVIEW

File: pkg/ako-operator/config_envvar_test.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
+// Copyright 2021 VMware, Inc.
```
Comment (XudongLiuHarold) [inline pkg/ako-operator/config_envvar_test.go:1]: nit: we are in 2022 now :)

File: pkg/ako-operator/suite_test.go [hunk 2]
```diff
@@ -0,0 +1,16 @@
+// Copyright 2020 VMware, Inc.
```
Comment (XudongLiuHarold) [inline pkg/ako-operator/suite_test.go:1]: nit: same here

**Review by XudongLiuHarold — APPROVED** [review]
Look good to me, just fix the file header.

/lgtm

File: pkg/ako-operator/config_envvar_test.go (hunk 1 above)
Comment (chenlin07) [reply pkg/ako-operator/config_envvar_test.go]: fixed. good catch!

Comment (chenlin07) [reply pkg/ako-operator/config_envvar_test.go]: btw. Should we update all files' headers to 2022?

Comment (XudongLiuHarold) [reply pkg/ako-operator/config_envvar_test.go]: We only need to update files we modified this year. :)

**Review by XudongLiuHarold — APPROVED** [review]
LGTM, thanks @chenlin07
